src/App.js

```diff
@@ -21,89 +25,95 @@ import theme from 'assets/theme';
 
 export default function App() {
   return (
-    <ThemeProvider theme={theme}>
-      <StylesProvider injectFirst>
-        <CssBaseline />
-        <BrowserRouter>
-          <AppBar />
-          <Switch>
-            <Route exact path="/" component={HomePage} />
+    <MuiThemeProvider theme={theme}>
+      <ScThemeProvider theme={theme}>
```

Comment:
This seems to be the recommended way to use a theme with both material-ui and styled-components. 😬 
https://github.com/mui-org/material-ui/issues/10098#issuecomment-574162533

(note: the rest of this file is unchanged; turn on "Hide whitespace changes" to make the diff much better)